ownCloudAppShared/Tools/AppStatistics.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  AppStatistics.swift
+//  ownCloudAppShared
+//
+//  Created by Michael Neuwert on 10.06.20.
+//  Copyright © 2020 ownCloud GmbH. All rights reserved.
+//
+
+/*
+ * Copyright (C) 2020, ownCloud GmbH.
+ *
+ * This code is covered by the GNU Public License Version 3.
+ *
+ * For distribution utilizing Apple mechanisms please see https://owncloud.org/contribute/iOS-license-exception/
+ * You should have received a copy of this license along with this program. If not, see <http://www.gnu.org/licenses/gpl-3.0.en.html>.
+ *
+ */
+
+import Foundation
+import ownCloudSDK
+import StoreKit
+
+extension TimeInterval {
+    public var days: Int {
+        return Int(self / (3600.0 * 24.0))
+    }
+}
+
+public class AppStatistics {
+
+    enum UserDefaultsKeys: String {
+        case launchDateDefaultsKey, updateDateDefaultsKey, lastVersionDefaultsKey, lastPromptedAboutReviewKey
```

Comment:
Since this is a `String` enum, the name of the keys in `UserDefaults` will also include the `Key` at the end, so the prefs file will look like:
```xml
<key>launchDateDefaultsKey</key><date>2020-06-12 12:34:56</date>
```
 As the enum is already called `UserDefaultsKeys`, I think the `Key` at the end can be safely removed.

Reply:
@felix-schwarz fixed